.github/workflows/extended.yml

```diff
@@ -67,11 +67,8 @@ jobs:
           fetch-depth: 1
       - name: Free Disk Space (Ubuntu)
         uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be
-      - name: Install Rust
-        run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-          source $HOME/.cargo/env
-          rustup toolchain install
+      - name: Setup Rust toolchain
```

Comment:
I think this will fail because (as I found out earlier in the week) the ubuntu builder doens't start with rust